app/src/main/java/com/github/factotum_sdp/factotum/ui/directory/DirectoryViewModel.kt

```diff
@@ -0,0 +1,14 @@
+package com.github.factotum_sdp.factotum.ui.directory
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+class DirectoryViewModel : ViewModel() {
+
+    private val _text = MutableLiveData<String>().apply {
+        //temp text, let it hardcoded until true fragment implementation
```

Comment:
Should we use a TODO to find it more easily?

Reply:
I don't think it's needed here, as the Fragment class is empty and will be overridden by the developer working on the corresponding fragment"